### ISSUE TITLE
New window or changing into Private Mode sets keyboard focus into Search field

### ISSUE THREAD

**Issue by mpopp75** [Quicksaver/OmniSidebar#39]
When I open a new browser window or change into Private Mode, the keyboard focus is set into the sidebar's Search field. This behavior shouldn't change compared to the browser's default behavior, or at least there should be an option to retain the browser's default behavior.
